lua/nvim-treesitter/parsers.lua

```diff
@@ -285,6 +285,13 @@ list.ql = {
   maintainers = {'@pwntester'},
 }
 
+list.verilog = {
+  install_info = {
+    url = "https://github.com/tree-sitter/tree-sitter-verilog",
+    files = { "src/parser.c" },
+  },
+  filetype = 'systemverilog',
```

Comment:
Is there a way to add multiple filetypes? This parser works for both verilog and systemverilog.

Reply:
There is used_by. Like with c++ and vida.

Reply:
fixed it